backend/siarnaq/bracket/challonge.py

```diff
@@ -322,15 +322,13 @@ def get_match_and_participant_objects_for_round(round: TournamentRound):
         )
         match_objects.append(match_object)
 
-        # Note that Challonge 1-indexes its player indexes
-        # while our internal data model (in Siarnaq) 0-indexes.
-        for (siarnaq_player_index, challonge_player_index) in enumerate(
-            ["player1", "player2"]
-        ):
+        for player_index in range(2):
```

Comment:
```suggestion
        for (player_index, challonge_participant) in enumerate(
            challonge_match["attributes"]["points_by_participant"]
        ):
```

removes magic number `2` by lifting stuff up from l.327